scalding-core/src/main/scala/com/twitter/scalding/filecache/DistributedCacheFile.scala

```diff
@@ -105,16 +105,14 @@ final case class UncachedFile private[scalding] (source: Either[String, URI]) {
   }
 
   private[this] def addHdfs(conf: Configuration): CachedFile = {
-    HDistributedCache.createSymlink(conf)
```

Comment:
I guess we need to find out if this method ever did anything. Maybe it never did.